client/web/src/search/results/SearchResultsInfoBar.tsx

```diff
@@ -64,55 +70,79 @@ const QuotesInterpretedLiterallyNotice: React.FunctionComponent<SearchResultsInf
-                    wrapInList={false}
-                    showLoadingSpinnerDuringExecution={true}
-                    actionItemClass="btn btn-link nav-link text-decoration-none"
-                />
-
-                {props.resultsFound && (
-                    <li className="nav-item">
-                        <button
-                            type="button"
-                            onClick={props.onExpandAllResultsToggle}
-                            className="btn btn-link nav-link text-decoration-none"
-                            data-tooltip={`${props.allExpanded ? 'Hide' : 'Show'} more matches on all results`}
-                        >
-                            {props.allExpanded ? (
-                                <>
-                                    <ArrowCollapseVerticalIcon className="icon-inline" /> Collapse all
-                                </>
-                            ) : (
-                                <>
-                                    <ArrowExpandVerticalIcon className="icon-inline" /> Expand all
-                                </>
-                            )}
-                        </button>
-                    </li>
-                )}
-
-                {props.showSavedQueryButton !== false && props.authenticatedUser && (
-                    <li className="nav-item">
-                        <button
-                            type="button"
-                            onClick={props.onSaveQueryClick}
-                            className="btn btn-link nav-link text-decoration-none"
-                        >
-                            <DownloadIcon className="icon-inline test-save-search-link" /> Save this search query
-                        </button>
-                    </li>
-                )}
-            </ul>
-        </small>
-    </div>
-)
+                <ul className="nav align-items-center justify-content-end">
+                    <ActionsNavItems
+                        {...props}
+                        extraContext={{ searchQuery: props.query || null }}
+                        menu={ContributableMenu.SearchResultsToolbar}
+                        wrapInList={false}
+                        showLoadingSpinnerDuringExecution={true}
+                        actionItemClass="btn btn-link nav-link text-decoration-none"
+                    />
+                    {CreateCodeMonitorButton}
+                    {props.showSavedQueryButton !== false && props.authenticatedUser && (
+                        <li className="nav-item">
+                            <button
+                                type="button"
+                                onClick={props.onSaveQueryClick}
+                                className="btn btn-link nav-link text-decoration-none"
+                            >
+                                <DownloadIcon className="icon-inline test-save-search-link" /> Save this search query
```

Comment:
```suggestion
                                <DownloadIcon className="icon-inline test-save-search-link" /> Save search
```

For brevity and to recapture some space.

Reply:
Thanks! Applied.